modules/renter.go

```diff
@@ -335,9 +335,16 @@ type Renter interface {
 	// blocking, including downloads of `offset` and `length` type.
 	DownloadAsync(params RenterDownloadParameters) error
 
+	// ClearDownloadHistory clears the download history of the renter.
+	ClearDownloadHistory()
+
 	// DownloadHistory lists all the files that have been scheduled for download.
 	DownloadHistory() []DownloadInfo
 
+	// RemoveFromDownloadHistory removes a given download from the download
+	// history of the renter.
+	RemoveFromDownloadHistory(staticSiaPath string) error
```

Comment:
not good enough to remove every instance of downloading a file I think, I think we'd rather be able to remove a specific instance in the download history.

That means we probably need to give downloads in the history a unique identifier, probably a `fastrand` generated value

Reply:
@DavidVorick I was going to use the time stamp unless you'd prefer that i add a unique identifier